src/services/sticker.services.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import Constants from "expo-constants";
+
+const BASE_URL = Constants.expoConfig.extra.apiUrl;
+const URL_STICKERS_OBTAIN = `${BASE_URL}/stickers/obtain/1`; //numero depende del id del evento
```

Comment:
```suggestion
const URL_STICKERS_OBTAIN = `${BASE_URL}/stickers/obtain/1`; //numero depende del id del evento
```
Hmmm esta bien sus que eso este quemado, pero supongo quedara asi hasta que implementemos los multiples eventos, igual yo por ahora lo pondria en una constante nueva aqui en este mismo archivo para no quemarlo tan feo